reactor-netty-core/build.gradle

```diff
@@ -55,7 +55,12 @@ dependencies {
 	api "io.netty:netty-handler-proxy:$nettyVersion"
 	api "io.netty:netty-resolver-dns:$nettyVersion"
 	if (!"$nettyVersion".endsWithAny("SNAPSHOT")) {
-		api "io.netty:netty-resolver-dns-native-macos:$nettyVersion:osx-x86_64"
+		if (osdetector.classifier == "osx-x86_64" || osdetector.classifier == "osx-aarch_64") {
+			api "io.netty:netty-resolver-dns-native-macos:$nettyVersion$os_suffix"
+		}
+		else {
+			api "io.netty:netty-resolver-dns-native-macos:$nettyVersion:osx-x86_64"
```

Comment:
I wonder if that is still relevant if we cannot detect any `osx` from `osdetector` ?
ie should we default to pure `$nettyVersion` in that case ?

Reply:
@simonbasle Having the comment below, do we still need to address this?

Reply:
no, sorry, I answered as a main comment instead of resolving this conversion. go for merge.